pkgs/development/rocm-modules/6/miopen/default.nix

```diff
@@ -122,11 +136,11 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://github.com/ROCm/MIOpen/commit/3413d2daaeb44b7d6eadcc03033a5954a118491e.patch";
       hash = "sha256-ST4snUcTmmSI1Ogx815KEX9GdMnmubsavDzXCGJkiKs=";
     })
-    (fetchpatch {
-      name = "Extend-MIOpen-ISA-compatibility.patch";
-      url = "https://github.com/GZGavinZhao/MIOpen/commit/416088b534618bd669a765afce59cfc7197064c1.patch";
-      hash = "sha256-OwONCA68y8s2GqtQj+OtotXwUXQ5jM8tpeM92iaD4MU=";
-    })
+    # (fetchpatch {
```

Comment:
Missed this arch compat patch. Might cause missing kernels for some ops for GPUs that aren't officially supported like iGPUs.

Reply:
We should add a comment what to do with the patch in the future or remove it

Reply:
Related: arch compat patches might not be correct, gfx1031 MIOpen kernels can crash on gfx1030 according to https://github.com/ROCm/MIOpen/issues/3540

Reply:
A comment is still missing here (or removing the commented out patch)

Reply:
Commented